packages/shared/src/server/queues.ts

```diff
@@ -308,4 +341,16 @@ export type TQueueJobTypes = {
     payload: DeadLetterRetryQueueEventType;
     name: QueueJobs.DeadLetterRetryJob;
   };
+  [QueueName.WebhookQueue]: {
+    timestamp: Date;
+    id: string;
+    payload: WebhookInput;
+    name: QueueJobs.WebhookJob;
+  };
+  [QueueName.PromptVersionChangeQueue]: {
+    timestamp: Date;
+    id: string;
+    payload: PromptVersionChangeEventType;
+    name: QueueJobs.PromptVersionChangeJob;
+  };
```

Comment:
I see this already exploding in the number of queues. I would recommend a single EntityChange queue with an envelope message that indicates the body and then a type specific payload. Otherwise, this will become super messy to maintain IMO.